tezos_interop/dune

```diff
@@ -2,4 +2,13 @@
  (name tezos_interop)
  (libraries helpers lwt.unix mirage-crypto-ec tezos-micheline)
  (preprocess
-  (pps ppx_deriving_yojson)))
+  (pps ppx_deriving_yojson ppx_blob))
+ (preprocessor_deps
+  (file ./run_entrypoint.bundle.js)))
+
+(rule
+ (deps ./webpack.config.js ./run_entrypoint.js)
+ (targets ./run_entrypoint.bundle.js)
+ (mode fallback)
+ (action
+  (run webpack --env=%{profile})))
```

Comment:
Running webpack or any node binaries isn't well supported in esy. It's nice that running `esy` will be the JS artifacts too, but if it causes trouble, we'll have to move it outside the build environment (therefore, outside Dune too).

Great job though